ide/nativeimage.api/nbproject/project.xml

```diff
@@ -0,0 +1,111 @@
+                        <release-version>1</release-version>
+                        <specification-version>1.75</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>
+                    <code-name-base>org.openide.filesystems</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>9.18</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>
+                    <code-name-base>org.openide.util.lookup</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>8.41</specification-version>
+                    </run-dependency>
+                </dependency>
+            </module-dependencies>
+            <test-dependencies>
+                <test-type>
+                    <name>unit</name>
+                    <test-dependency>
+                        <code-name-base>org.netbeans.libs.junit4</code-name-base>
+                        <compile-dependency/>
+                    </test-dependency>
+                    <test-dependency>
+                        <code-name-base>org.netbeans.modules.cpplite.editor</code-name-base>
+                        <compile-dependency/>
+                    </test-dependency>
+                    <test-dependency>
+                        <code-name-base>org.netbeans.modules.masterfs</code-name-base>
+                        <compile-dependency/>
+                    </test-dependency>
+                    <test-dependency>
+                        <code-name-base>org.netbeans.modules.nbjunit</code-name-base>
+                        <recursive/>
+                        <compile-dependency/>
+                    </test-dependency>
+                    <test-dependency>
+                        <code-name-base>org.openide.util.ui</code-name-base>
+                        <compile-dependency/>
+                    </test-dependency>
+                </test-type>
+            </test-dependencies>
+            <friend-packages>
+                <friend>org.netbeans.modules.cpplite.debugger</friend>
+                <friend>org.netbeans.modules.gradle.java</friend>
+                <friend>org.netbeans.modules.java.lsp.server</friend>
+                <friend>org.netbeans.modules.java.nativeimage.debugger</friend>
+                <package>org.netbeans.modules.nativeimage.api</package>
+                <package>org.netbeans.modules.nativeimage.api.debug</package>
+                <package>org.netbeans.modules.nativeimage.spi</package>
+                <package>org.netbeans.modules.nativeimage.spi.debug</package>
+                <package>org.netbeans.modules.nativeimage.spi.debug.filters</package>
```

Comment:
When executing `ide/nativeimage.api$ ant javadoc` I don't get any Javadoc! Btw. don't forget to modify `nbbuild/build.properties` to include the Javadoc in the list of all NetBeans Javadocs.

Reply:
Fixed.